Zend/tests/attributes/007_wrong_compiler_attributes.phpt

```diff
@@ -0,0 +1,14 @@
+--TEST--
+attributes: Add PhpCompilerAttribute
```

Comment:
Scenario could need some clarification here

Reply:
Read the RFC again, and the scope of `PhpCompilerAttribute` is not clear: from a reader perspective, when would somebody encounter this?

Reply:
Userland will only see `PhpCompilerAttribute` when they call `ReflectionClass::getAttributes()` on an internal class that is also an attribute. This should rarely be the case, because you wouldn't use reflection on internal classes very often.